### DISCUSSION TITLE
Firefox plugin and Firefox Nightly

### DISCUSSION THREAD

**Discussion by tvalenta** [quicksilver/Quicksilver#2966] [Ideas]
I'd like to have the functionality of the Firefox Plugin works with Firefox Nightly, and am curious as to the best way to do this. The current plugin requires application bundle ID `org.mozilla.firefox`, but Firefox Nightly has the bundle ID of `org.mozilla.nightly`. Both applications use the same location for the Firefox Profile; would it be reasonable to extend the current plugin's code to work with both `org.mozilla.firefox` and `org.mozilla.nightly` or better done as a separate plugin?

**Comment by n8henrie** [comment]
I bet we can find a way to use the existing plugin. Will look into it.

**Reply by tvalenta** [reply]
I think I have a working mockup that I'm using myself, but I haven't tested it when both applications are installed. If nobody beats me to it, I'll get a PR going for review.
